index.js

```diff
@@ -58,6 +61,7 @@ try {
 
 const iastNativeMethods = {
   newTaintedString: addon.newTaintedString,
+  newTaintedObject: addon.newTaintedObject,
```

Comment:
Type declaration for the new method in [index.d.ts](https://github.com/DataDog/dd-native-iast-taint-tracking-js/blob/main/index.d.ts) file is missing

Reply:
https://github.com/DataDog/dd-native-iast-taint-tracking-js/pull/95